server/global.h

```diff
@@ -0,0 +1,25 @@
+#ifndef GLOBAL_H
+#define GLOBAL_H
+
+#include <cerrno>
+#include <cstdio>
+#include <cstring>
+#include <errno.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+// Next 3 stolen from csapp
+// ------------------ //
+ssize_t sio_put(char *msg);
```

Comment:
I don't think we should be using Linux syscalls for terminal read/write in the scope of this project. Let's use iostream for that. All these functions can be deleted. Also, we probably do not need to wrap the call to _exit() to a separate error function that's only writing the error message to STDOUT and closes the child.